client/jetpack-cloud/sections/partner-portal/company-details-form/components/searchable-dropdown.tsx

```diff
@@ -0,0 +1,55 @@
+import { Gridicon } from '@automattic/components';
+import Select, { StylesConfig, DropdownIndicatorProps, components } from 'react-select';
+
+interface Props {
+	className?: string;
+	value: object;
+	options: object[];
+	onChange: ( option: any ) => void;
+	isDisabled?: boolean;
+	isLoading?: boolean;
+}
+
+const colourStyles: StylesConfig = {
+	control: ( styles ) => ( {
+		...styles,
+		height: 40,
+		borderRadius: 2,
+		border: '1px solid var(--color-neutral-10)',
+		color: 'green',
+		padding: '0 8px 0 4px',
+	} ),
+	singleValue: ( styles ) => ( { ...styles, color: 'var(--color-neutral-70)', fontWeight: 600 } ),
```

Comment:
@kallehauge, I purposely let it as it was before. I didn't want to change it since I presumed it was approved by someone in the past. This is a screenshot of how it looks in the `trunk` branch.

![image](https://user-images.githubusercontent.com/6406071/198070488-267ec2b6-2ed4-4211-8dda-1c680cf82385.png)